src/types/jsonData.ts

```diff
@@ -1,12 +1,14 @@
-type JSONValue =
-    | string
-    | number
-    | boolean
-    | JSONObject
-    | JSONArray;
+/* eslint-disable no-use-before-define */
```

Comment:
this does the beg question whether this should be kept if it's unused.  This has come from the web-starter project. 
Are we going to be using raw JSON in the web?

Reply:
file removed